src/compound/lens/LensExtension.sol

```diff
@@ -0,0 +1,192 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity 0.8.13;
+
+import "../interfaces/IRewardsManager.sol";
+import "../interfaces/IMorpho.sol";
+import "./interfaces/ILensExtension.sol";
+
+import "@morpho-dao/morpho-utils/math/CompoundMath.sol";
+
+/// @title LensExtension.
+/// @author Morpho Labs.
+/// @custom:contact security@morpho.xyz
+/// @notice This contract is an extension of the Lens.
+contract LensExtension is ILensExtension {
```

Comment:
We don't add this contract to the storage check, because it does not have any storage right ?